generators/server/templates/src/main/resources/config/application-dev.yml.ejs

```diff
@@ -143,30 +143,11 @@ spring:
             enabled: false
     <%_ } _%>
     jpa:
-        <%_ if (devDatabaseType === 'mysql') { _%>
-        database-platform: org.hibernate.dialect.MySQL8Dialect
-        <%_ } else if (devDatabaseType === 'mariadb') { _%>
-        database-platform: org.hibernate.dialect.MariaDB103Dialect
-        <%_ } else if (devDatabaseType === 'postgresql') { _%>
+        <%_ if (devDatabaseType === 'postgresql') { _%>
         database-platform: io.github.jhipster.domain.util.FixedPostgreSQL10Dialect
-        <%_ } else if (devDatabaseType === 'oracle') { _%>
-        database-platform: org.hibernate.dialect.Oracle12cDialect
-        <%_ } else if (devDatabaseType === 'mssql') { _%>
-        database-platform: org.hibernate.dialect.SQLServer2012Dialect
-        <%_ } else { _%>
+        <%_ } else if (devDatabaseType === 'h2disk' || devDatabaseType === 'h2Memory') { _%>
         database-platform: io.github.jhipster.domain.util.FixedH2Dialect
         <%_ } _%>
-        <%_ if (devDatabaseType === 'mysql' || devDatabaseType === 'mariadb') { _%>
-        database: MYSQL
-        <%_ } else if (devDatabaseType === 'postgresql') { _%>
-        database: POSTGRESQL
-        <%_ } else if (devDatabaseType === 'oracle') { _%>
-        database: ORACLE
-        <%_ } else if (devDatabaseType === 'mssql') { _%>
-        database: SQL_SERVER
-        <%_ } else { _%>
-        database: H2
```

Comment:
Could this be also removed from `test/application.yml`?

Reply:
Not sure it is possible, because we might have a different database in dev and prod

Reply:
@hdurix : good catch. I'll remove the `spring.jpa.database` entry too

Reply:
Forget, a esse thinking about main/application.yml 🤪😅